samples/snippets/src/main/java/com/example/bigquery/AuthorizeDataset.java

```diff
@@ -58,12 +58,12 @@ public static void authorizeDataset(DatasetId sourceDatasetId, DatasetId userDat
           new Acl.DatasetAclEntity(userDatasetId, targetTypes);
       sourceDatasetAcl.add(Acl.of(userDatasetAclEntity));
 
-      // update the user dataset with source dataset's ACL
-      Dataset updatedUserDataset =
-          userDataset.toBuilder().setAcl(sourceDatasetAcl).build().update();
+      // update the source dataset with user dataset's ACL
```

Comment:
The method comment says: "This method will update userDataset's ACL with sourceDataset's ACL". Does this need to be updated as well or is this the right change?

Reply:
Hi @averikitsch Thanks for pointing out ! 
I have updated the method. 
And yes, in this case, source dataset ACL should be updated (a similar case for authorizing a view can be found here https://github.com/googleapis/java-bigquery/blob/HEAD/samples/snippets/src/main/java/com/example/bigquery/AuthorizedViewTutorial.java#L99 )